Koha/Schema/Result/InvoiceAdjustment.pm

```diff
@@ -0,0 +1,155 @@
+use utf8;
+package Koha::Schema::Result::InvoiceAdjustment;
+
+# Created by DBIx::Class::Schema::Loader
+# DO NOT MODIFY THE FIRST PART OF THIS FILE
+
+=head1 NAME
+
+Koha::Schema::Result::InvoiceAdjustment
+
+=cut
+
+use strict;
+use warnings;
+
+use base 'DBIx::Class::Core';
+
+=head1 TABLE: C<invoice_adjustments>
+
+=cut
+
+__PACKAGE__->table("invoice_adjustments");
+
+=head1 ACCESSORS
+
+=head2 adjustment_id
+
+  data_type: 'integer'
+  is_auto_increment: 1
+  is_nullable: 0
+
+=head2 invoiceid
+
+  data_type: 'integer'
+  is_foreign_key: 1
+  is_nullable: 0
+
+=head2 adjustment
```

Comment:
I'm not generally a fan of having column names match table names match class names.. you have much more flexibility and clarity if you keep your names more descriptive.. I'd use 'amount'.. then you're calling constructs are `adjustement->amount` rather than `->adjustment->adjustment` for example.